jenkins/bootstrap.py

```diff
@@ -832,13 +832,13 @@ def bootstrap(args):
     exc_type = None
 
     try:
+        setup_magic_environment(job)
```

Comment:
Yeah that will work!

What do you think of changing things to `--ssh=.ssh/id_ed25519` aka we store the value of the var we should set? I'm not sure if that is better or too annoying.